pymc3/sampling.py

```diff
@@ -327,6 +329,10 @@ def sample(
         is drawn from.
 
         Sampling can be interrupted by throwing a ``KeyboardInterrupt`` in the callback.
+    return_inferencedata: bool
```

Comment:
minor points:

1. the `return_inferencedata` should be `bool, optional`
2. this format for docstrings doesn't comply with the numpy format, which has spaces on both sides of the ":"

https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Fixed in my edit commit.

Reply:
thanks 👍

Reply:
I think I'm the one who deleted the first space before the ":" in a previous PR -- this was because when you double-click on an arg on the website, this annoyingly (and weirdly) selects the arg, the colon and the first word after the colon. 
When the first space is not there (like: `arg: details`), double-clicking only selects the arg, as it should. See [this page](https://arviz-devs.github.io/arviz/generated/arviz.plot_compare.html#arviz.plot_compare) of ArviZ doc for instance, for an example of both (double-click on `figsize` and then on `textsize`). 
So I'm in favor of keeping the previous version, if this doesn't have any negative side effects.

Reply:
I'm afraid it *does* have a negative effect: Sphinx will not format correctly docstrings that don't have " : ". 

I had put in some of these ": " docstrings myself, but I'm afraid they are wrong.  See the numpy doc I linked above.

Reply:
I have no opinion on this. What does our doc-master, @ColCarroll say?

Reply:
I'll wait for @ColCarroll to weigh in, but I will point out that we agreed to use numpy style, and the numpy style description is quite clear about what it expects.

Here's an example of what Sphinx does wrong if  you don't put the spaces around the colons -- the variable names and types run together:

![image](https://user-images.githubusercontent.com/3274/80976610-443d8400-8de9-11ea-97c6-b450f76c4390.png)